oeps/oep-0011-bp-FED-technology.rst

```diff
@@ -48,33 +48,42 @@ Specification
 Technology Selection
 ~~~~~~~~~~~~~~~~~~~~
 
-1. **Server-side content should be rendered with Django Templates**
+1. **Use React and Redux**
 
-   **Rationale**: There are many template languages available for Django,
-   but the simplest option is to use the built-in Django template engine.
-   The Open edX codebase has a mixture of Django and Mako templates, but the
-   former are easier to reason about because they don't support arbitrary
-   code evaluation. See `Reasons for rejecting Mako`_ for a more detailed
-   explanation.
+   **Rationale**: React should be used for building new UIs, as it is
+   widely adopted by the community and strikes a balance between
+   flexibility and feature richness. For state management of complex
+   client-side interactions, Redux should be used. This library was chosen
+   because it sees strong use in the React community, but is also flexible
+   enough to be used in situations where a hybrid React/Backbone architecture
+   exists.
 
-   **Exception**: Mako templates can continue to be used within edx-platform
-   for consistency with the existing code base. This is because the base
-   templates and theming support are all provided via Mako, so it is too
-   much to expect a new feature to be implemented with a different framework.
-   There is much desire to replace Mako within edx-platform so this
-   exception may eventually be removed.
+   When building in existing Django server-rendered pages, one can use the
+   react_render helper method to bridge to React. This bridge provides an
+   easy way to pass data into a React component from Django via props. For
+   more information about react_render see `ReactRender`_.
+
+   React also has a strong testing story. React components can be tested in
+   isolation with unit tests using `Jest`_ and `Enzyme`_.
```

Comment:
Not sure how far we want to go down the testing rabbit-hole here. We could also mention we are evaluating cypress for e2e, but given it is not widely adopted yet at edX I'm not sure it belongs here.